data/core/units/drakes/Arbiter.cfg

```diff
@@ -169,4 +170,8 @@
             image="units/drakes/arbiter.png:100"
         [/frame]
     [/attack_anim]
+    [female]
+        name= _ "Drake Arbiter"
```

Comment:
These all be `female^Drake XYZ`, otherwise they won't appear as separate strings for translation.

Reply:
Good catch, thanks! I used the naga fighter as a starting point but that actually replaces the -a suffix with a (Sanskrit?) -ini for its female variant.